src/cli/checkout/deploy.ts

```
@@ -122,6 +86,84 @@ export const handler = async (argv: Arguments<Options & { name: string }>) => {
   process.exit(0);
 };
 
+export const setupSaleorAppCheckout = async (
+  vercelToken: string,
+  appName: string,
+  url: string,
+  vercel: Vercel,
+  argv: Arguments<Options>
+) => {
+  // SETUP CHECKOUT APP
+  await createCheckoutApp(vercelToken, appName, url);
+  const checkoutAppDeploymentId = await deployVercelProject(
+    vercelToken,
+    appName
+  );
+  await vercel.verifyDeployment(appName, checkoutAppDeploymentId);
+  const { alias } = await vercel.getDeployment(checkoutAppDeploymentId);
```

Response from external system should be always validated. `alias` field can be missing. Let's check it's existence and fail early with proper message